### PR TITLE
Add banner to arbitrum sepolia testnet

### PR DESCRIPTION
**Pull request by guilhermebins**
This enhancement adds the Arbitrum Sepolia testnet banner to be displayed when users are connected.

### CODE REVIEW

**Comment by pikonha** [comment]
we really appreciate your contribution to this @guilhermebins ✨ 

although the issue mentions adding the banner for `arbitrum sepolia`, I'd advise relying on the [viem's chain definition](https://github.com/wevm/viem/blob/main/src/chains/definitions/arbitrumGoerli.ts) which included a `testnet` flag for each chain.

the condition would be like this then: 

```ts
 const isConnectedToTestnet = () => chain?.testnet;
```

File: components/01-atoms/DappHeader.tsx [hunk 1]
```diff
@@ -36,8 +36,11 @@ export const DappHeader = () => {
   const router = useRouter();
   const currentRoute = router.pathname;
 
+  const testnetChainIds: number[] = [sepolia.id,
+                                     arbitrumSepolia.id,
+                                     /*add other testnet chain.id here*/];
```
Comment (FrancoAguzzi) [inline components/01-atoms/DappHeader.tsx:41]:
```suggestion
  const testnetChainIds: number[] = [sepolia.id, arbitrumSepolia.id];
```

**Review by FrancoAguzzi — CHANGES_REQUESTED** [review]

File: components/01-atoms/DappHeader.tsx (hunk 1 above)
Comment (FrancoAguzzi) [reply components/01-atoms/DappHeader.tsx:41]: Hey @guilhermebins!

One comment on this, can you address it?

**Review by eduramme — APPROVED** [review]

**Review by FrancoAguzzi — APPROVED** [review]

**Comment by FrancoAguzzi** [comment]
@guilhermebins thank you, looks sharp 📐✨

**Comment by FrancoAguzzi** [comment]
Captura de Tela 2024-10-04 às 14 48 49

Captura de Tela 2024-10-04 às 14 48 39

**Pull request merged**